app/containers/pageContainers/UserOrganisationFacilities.js

```diff
@@ -0,0 +1,37 @@
+import React, {Component} from 'react';
+import {Container} from 'react-bootstrap';
+import {graphql} from 'react-relay';
+import DefaultLayout from '../../layouts/default-layout';
+import OrganisationFacilities from '../Organisations/OrganisationFacilities';
+
+export default class UserOrganisationFacilities extends Component {
+  static query = graphql`
+    query UserOrganisationFacilitiesQuery($id: ID!) {
+      query {
+        ...OrganisationFacilities_query @arguments(id: $id)
+      }
+    }
+  `;
+
+  render() {
+    const {router} = this.props;
+    const orgTitle = `${router.query.organisationName} Facilities`;
```

Comment:
Should the org name come from relay instead of the query string?

Reply:
yes!